### DISCUSSION TITLE
How to delete user app rows

### DISCUSSION THREAD

**Discussion by Kry9toN** [Budibase/budibase#14052] [Help]
How to fix user duplicate like this and delete the rows?

![image](https://github.com/Budibase/budibase/assets/44697929/318a8216-16f7-4c82-85b0-8267eb102844)

**Comment by ConorWebb96** [comment]
I don't actually think this is possible unless you were manually manipulating the users table which you shouldn't be doing. That is a read only table.

The quickest way to resolve what you are seeing within the users table would be to export your app and then re-import it again.

I hope that helps.

**Reply by Kry9toN** [reply]
table on couchdb?

**Reply by Kry9toN** [reply]
how about editing database table?
